test/go.mod

```diff
@@ -0,0 +1,100 @@
+	github.com/golang/protobuf v1.3.5 // indirect
+	github.com/google/go-cmp v0.4.1 // indirect
+	github.com/google/go-containerregistry v0.0.0-20200331213917-3d03ed9b1ca2 // indirect
+	github.com/google/gofuzz v1.1.0 // indirect
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/googleapis/gnostic v0.4.0 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/json-iterator/go v1.1.9 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/mailru/easyjson v0.7.1-0.20191009090205-6c0755d89d1e // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
+	github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
+	github.com/openzipkin/zipkin-go v0.2.2 // indirect
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200326003131-5ff376fd8c14
+	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.4.2 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	go.opencensus.io v0.22.3 // indirect
+	go.uber.org/atomic v1.6.0 // indirect
+	go.uber.org/multierr v1.5.0 // indirect
+	go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee // indirect
+	go.uber.org/zap v1.14.1 // indirect
+	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073 // indirect
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775 // indirect
+	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/tools v0.0.0-20200329025819-fd4102a86c65 // indirect
+	gomodules.xyz/jsonpatch v2.1.0+incompatible // indirect
+	google.golang.org/appengine v1.6.5 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.3.0 // indirect
+	honnef.co/go/tools v0.0.1-2020.1.3 // indirect
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
+	k8s.io/klog v1.0.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
+	k8s.io/utils v0.0.0-20200124190032-861946025e34 // indirect
+	knative.dev/eventing v0.15.4
+	knative.dev/operator v0.14.3-0.20200605190218-43fce8e820ed
+	knative.dev/pkg v0.0.0-20200625173728-dfb81cf04a7c
+	knative.dev/serving v0.15.3
+	knative.dev/test-infra v0.0.0-20200519161858-554a95a37986
+	sigs.k8s.io/yaml v1.2.0 // indirect
+)
+
+replace (
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200527184302-a843dc3262a0
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
+
+	// Pin to kube 1.16
+	k8s.io/api => k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
```

Comment:
Can we use actual tags here, that are readable? Do we need to pin all of those?

Reply:
I couldn't get it working without the pins (which came from olm itself)

https://github.com/operator-framework/operator-lifecycle-manager/blob/release-4.3/go.mod#L54

Reply:
keep in mind -- it's not serving that made this a royal pain, it was adding eventing that brought in a dependency that wasn't compatible with go mod